admin/controller/altapay/templates/admin/controller/altapay.twig.php

```diff
@@ -232,6 +232,11 @@ public function order()
                     if ($charge <= 0) {
                         $charge = 0;
                     }
+                    if (isset($pay->ReconciliationIdentifiers) and !empty($pay->ReconciliationIdentifiers)) {
+                        foreach ($pay->ReconciliationIdentifiers as $reconciliation_identifier){
+                            $reconciliation_identifiers[$reconciliation_identifier->Id] = $reconciliation_identifier->Type;
+                        }
+                    }
```

Comment:
It seems that you are calculating the reconciliation identifiers on the spot, every time an order is being opened.

Why not storing the reconciliation identifiers in the order, and loading them from the database when the order is loaded?